vkcs/internal/clients/config.go

```diff
@@ -205,6 +209,24 @@ func (c *configer) MLPlatformV1Client(region string) (*gophercloud.ServiceClient
 	return client, err
 }
 
+func (c *configer) CapabilityV1Client(region string) (*gophercloud.ServiceClient, error) {
+	client, err := c.CommonServiceClientInit(newCapabilityV1, region, "ics")
+	if err != nil {
+		return client, err
+	}
+
+	return client, icapabilities.HealthCheck(client).Err
```

Comment:
```suggestion
	if err := capabilities.HealthCheck(client).ExtractErr(); err != nil {
	     return err
	}
	
	return client, nil
```

Reply:
done

Reply:
We don't usually refer to attributes of returned result structure, as there should be method that allows to represent it as an error.

Reply:
Should I create my own structure for the HelthCheck and ListUsers? gophercloud.Result does not have such method

Reply:
You should already have structure, don't you? Does `HealthCheck` have response body at all? If it doesn't than you should embed `gophercloud.ErrResult` instead. For `ListUsers` you can extract it as usual.

Reply:
Added two structures